thrift-confignode/src/main/thrift/confignode.thrift

```diff
@@ -117,6 +117,11 @@ struct TAuthorizerReq {
   7: required string nodeName
 }
 
+struct TAuthorizerResp {
+    1: required common.TSStatus status
+    2: required map<string, list<string>> authorizerInfo
+}
+
```

Comment:
I suggest using a dataset definition that is more common. Better to discuss with the query guys.

Reply:
Here I have used PermissionInfoDataSet to store the found information

Reply:
I see, but since the permission queries are shown to clients in a very similar way as data queries, I doubt is it necessary to use a separate structure.

Reply:
But currently everyone is using a separate structure

Reply:
Here is the result of the query returned from confignode to datanode, this structure should be ok

Reply:
I asked here, it doesn't matter if it is not unified, because after finally reaching the Coordinator, when the Coordinator returns to the client, it will be converted into a unified structure